src/components/ImageCard/ImageHotspots.jsx

```diff
@@ -472,9 +484,13 @@ const ImageHotspots = ({
 
   const imageStyle = {
     cursor: isEditable && !dragging ? 'crosshair' : 'auto',
-    position: 'relative',
-    left: image.offsetX,
-    top: image.offsetY,
+    position: 'absolute',
```

Comment:
I think these changes break the `onAddHotspotPosition` but unfortunately the test didn't seem to break. Check the story and notice the incorrect values of the action callback when adding a new hotspot..
https://deploy-preview-1824--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-imagehotspots--editable

Reply:
Great catch. I wouldn't have noticed that at all.